dev/Scroller/Scroller.idl

```diff
@@ -1,5 +1,14 @@
 ﻿[WUXC_VERSION_PREVIEW]
 [webhosthidden]
+enum ContentOrientation
+{
+    Vertical = 0,
+    Horizontal = 1,
+    None = 2,
```

Comment:
Is it more reasonable to set None=0 and make it before Vertical?

Reply:
I will mention that to Micah when he comes back. The spec is as above because Vertical is the default value for ScrollViewer.